### PR TITLE
Cleanup script for unused entities in the whisks database.

### PR DESCRIPTION
**Pull request by steffenrost**
Python script for cleanup that runs over all documents in the whisks database and checks if its namespace  still exists. The document will be marked for deletion or deleted if already marked for certain period of time if its namespace does not exists.

### CODE REVIEW

**Comment by rabbah** [comment]
@lue-kle can you please provide a description for your pull request.

**Comment by lue-kle** [comment]
Playground3 (Build #1966) run successfully on this fork/branch.

**Comment by rabbah** [comment]
@lue-kle do you have an Apache CLA?

File: tools/db/cleanUpWhisks.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        self.data = []
+        self.maxsize = size * 2
+
+    def append(self, ns, bool):
+
+        self.index=(self.index+2)%self.maxsize
+
+        if len(self.data) < self.maxsize:
+            self.data.append(ns)
+            self.data.append(bool)
```
Comment (markusthoemmes) [inline tools/db/cleanUpWhisks.py]: Python supports tuples. Should we store tuples of (String, Boolean) instead of relying on a fixed order here?

**Comment by rabbah** [comment]
As a general note: It would be useful in the long run to consolidate this and scripts like it into wskadmin (and rearchitect that accordingly).

**Comment by lue-kle** [comment]
@rabbah yes I have signed the CLA. I will talk to @cbickel regarding your suggestion for consolidation.

File: tools/db/cleanUpWhisks.py (hunk 1 above)
Comment (lue-kle) [reply tools/db/cleanUpWhisks.py]: sure we can use tuples, would than require two checks if the ns exists in the cache. I would suggest to do this change when adapting the script to be used also for kafka and cloudant trigger dbs.

**Pull request merged**